controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -254,7 +254,7 @@ def test_create_snapshot_raise_snapshot_source_pool_mismatch(self):
         xcli_volume = self._get_cli_volume()
         self.mediator.client.cmd.vol_list.return_value = Mock(as_single_element=xcli_volume)
         with self.assertRaises(array_errors.SnapshotSourcePoolMismatch):
-            self.mediator.create_snapshot(snapshot_volume_wwn, snapshot_name, "different_pool")
+            self.mediator.create_snapshot(snapshot_volume_wwn, snapshot_name, None, "different_pool")
```

Comment:
I would use named arguments for hardcoded values

Reply:
done

Reply:
does it also work with [assert_called_once_with](https://github.com/IBM/ibm-block-csi-driver/blob/f77a552c283383cab8f7dc733326f60c94940223/controller/tests/controller_server/csi_controller_server_test.py#L249)?

Reply:
nope